### DISCUSSION TITLE
Can't find the document anymore

### DISCUSSION THREAD

**Discussion by achievecreative** [vercel/next.js#68393] [Help]
### Summary

I remembered there were some pages to describe the difference between dev and production mode, especially for page routes.
But I can't find it any more, Can someone share the link if have.

Why remove these documents, are they not valid anymore?

Thanks in advance.

**Comment by icyJoseph** [comment]
Is it this one? https://nextjs.org/docs/pages/building-your-application/deploying/production-checklist 

You can also use the CN domain docs, and translate them to English in your browser, https://www.nextjs.cn/docs/basic-features/pages
